hrs-portlets-webapp/src/main/webapp/WEB-INF/jsp/contactInfo.jsp

```diff
@@ -227,21 +227,13 @@
 
   <sec:authorize ifAnyGranted="ROLE_UW_EMPLOYEE_ACTIVE">
   <c:choose>
-    <c:when test="${not empty prefs['updateMyPersonalInfoUrl']
-      && not empty prefs['updateMyPersonalInfoUrl'][0]}">
-
-      <!-- configured with HRS self-service personal information update
-       URL via portlet-preference. Use HRS PUM 22 content. -->
-
-       <!-- federal reporting statuses deliberately removed in PUM22 mode,
-            in favor of that data just being down inside
-            linked HRS self-service UI. -->
+    <c:when test="${not empty hrsUrls['Personal Information']">
```

Comment:
Missing `}` fixed on `uw-master` in https://github.com/UW-Madison-DoIT/hrs-portlets/commit/4872071ad9b71d0bd4d6baeef9f02f619dc7995d .